src/js/components/AppListComponent.jsx

```diff
@@ -186,7 +187,10 @@ var AppListComponent = React.createClass({
 
     if (filterText != null && filterText !== "") {
       nodesSequence = nodesSequence
-        .filter(app => app.id.indexOf(filterText) !== -1);
+        .filter(app => {
+          return score(app.id, filterText) > 0.025;
```

Comment:
But what sense? 
And what else is hidden then unexpected?
Cannot detect the use case for the scoring here.